django/thunderstore/repository/views/repository.py

```diff
@@ -508,7 +517,7 @@ def get(self, *args, **kwargs):
             PackageListing,
             package__owner__name=owner,
             package__name=name,
-            community=self.request.community,
+            community__identifier=self.community_identifier,
```

Comment:
`community=self.community` given that it should be accessible in `CommunityMixin` and is a more performant query (since `__identifier` adds a table join)